File: data/code_pairs/fortran/64.f90
module pr94329_mod
  implicit none
contains
  subroutine pr94329 (s, t)
    real :: s
    real, dimension(:,:) :: t
    integer :: i, j
    do i = 1,3
      do j = 1,3
        s = t(i,j)
      end do
    end do
  end subroutine pr94329
end module pr94329_mod

File: data/code_pairs/cpp/64.cpp
#include <array>
#include <iostream>

void pr94329(float& s, std::array<std::array<float, 3>, 3>& t) {
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            s = t[i][j];
        }
    }
}